lib/integrations/CalDav/CalDavCalendarAdapter.ts

```diff
@@ -229,11 +235,13 @@ export class CalDavCalendar implements CalendarApiAdapter {
 
       return calendars
         .filter((calendar) => {
-          return calendar.components.includes("VEVENT");
+          return calendar.components && calendar.components.includes("VEVENT")
```

Comment:
I think this evaluates to always be true; I checked in the console to verify [] is also truthy.

```javascript
calendar.components?.includes("VEVENT") // instead maybe?
```

Reply:
It's a check on null or undefined, but ?.includes is cleaner regardless.